src/App/app.test.js

```diff
@@ -1,5 +1,23 @@
-import { sum } from './sum';
+import React from 'react';
+import App from './';
+import { mount, shallow } from 'enzyme';
+import { Button } from '@patternfly/react-core';
 
-test('adds 1 + 2 to equal 3', () => {
-  expect(sum(1, 2)).toBe(3);
+describe('App tests', () => {
+  test('should render default App component', () => {
+    const view = shallow(<App />);
+    expect(view).toMatchSnapshot();
+  });
+
+  it('should render a dismiss button', () => {
+    const wrapper = mount(<App />);
+    expect(wrapper.find(Button)).toBeDefined();
```

Comment:
This is kind of a false positive. The value returned by wrapper.find will always be defined. If the element does not exist it returns a wrapper with length 0. You can use
expect(wrapper.find(Button).exists()).toBe(true)